### DISCUSSION TITLE
Amazing stuff!

### DISCUSSION THREAD

**Discussion by rookste** [Afischbacher/Nhl.Api#63] [General]
Awesome! Awesome! Awesome!

This is just fantastic! Well done sir!

**Comment by Afischbacher** [comment]
Hey @rookste 

Thanks for the appreciation, please let me know if you have any questions, suggestions or improvements 👍

**Comment by rookste** [comment]
No probs!

The solution is structured so nicely. Super well commented, documented and tested. I'd love to help if you need it. I'm not exactly sure what the NHL has to offer as far as advanced stats like Goal%, xGoal%,, Shot Attempts % (Corsi) and Unblocked Shot Attempts % (Fenwick). I really like the filter expressions!  Thanks for starting this project!

Cheers, steve
